src/main/resources/feedbackQuestionRankResultStatsRecipientTemplateSelfResponse.html

```diff
@@ -0,0 +1,32 @@
+<div class="panel-body">
+    <div class="row">
+        <div class="col-sm-4 text-color-gray">
+            <strong>
+                Response Summary
+            </strong>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm-12">
+            <table class="table table-bordered table-responsive margin-0">
+                <thead>
+                <tr>
+                    <td class="button-sort-none" onclick="toggleSort(this);">Team
+                        <span class="icon-sort unsorted"></span></td>
+                    <td class="button-sort-ascending" id="button_sortteamname" onclick="toggleSort(this);" style="width: 35%;">${optionRecipientDisplayName}
+                        <span class="icon-sort unsorted"></span></td>
+                    <td class="button-sort-none" id="button_sortname" onclick="toggleSort(this);" style="width:15%;">Ranks Received
+                        <span class="icon-sort unsorted"></span></td>
+                    <td class="button-sort-none" id="button_sortclaimed" onclick="toggleSort(this);" style="width:15%;">Average Rank
+                        <span class="icon-sort unsorted"></span></td>
+                    <td class="button-sort-none" id="button_sortrank" onclick="toggleSort(this);" style="width:15%;">Average Rank Excluding Self
+                        <span class="icon-sort unsorted"></span></td>
+                </tr>
```

Comment:
I'm aware that this is a mistake from an existing template, but let's correct these:
```html
<thead>
    <tr>
        <td class="button-sort-none toggle-sort">
            Team <span ...></span>
        </td>
        ...
    </tr>
</thead>
```
In addition, we decommission the usage of `toggleSort` in `onclick` some time back. Use `toggle-sort` class instead and it will still work.